Extensions/xkit_preferences.css

```diff
@@ -1542,7 +1542,7 @@
 .xkit--react #xkit-gallery-extension-tagviewer,
 .xkit--react #xkit-gallery-extension-video_downloader,
 .xkit--react #xkit-gallery-extension-read_more_now,
-.xkit--react #xkit-gallery-extension-retags,
+/* .xkit--react #xkit-gallery-extension-retags, (retags is mostly broken but works on blog pages) */
```

Comment:
no comments! this PR (And linked discord discussion) is clear enough documentation of this.